### PR TITLE
chore: Add Social Strategy and Event agenda

### PR DESCRIPTION
**Pull request by onedesert**

### CODE REVIEW

File: guidelines/monthly.md [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
+- Networking with drinks and food 20m - 1h
+
+## Sponsor requirements:
+
+- Sponsor must provide the information 3 weeks before the event.
+- Sponsor must provide company name
+- Sponsor must provide venue address. **Ideally, Google Maps link**
+- Sponsor must indicate venue capacity.
+- Sponsor should sponsor food/drinks.
+- Sponsor should provide talk brief and speaker Twitter/LinkedIn
```
Comment (dmitry-zaets) [inline guidelines/monthly.md:37]:
```suggestion
- Sponsor should provide at least one speaker (English speaking)
- Sponsor should provide talk brief and speaker Twitter/LinkedIn
```

File: guidelines/monthly.md [hunk 2]
```diff
@@ -1,27 +1,47 @@
...
+
+## Social strategy
+
+- Post Meetup event (🚧 create template and link to it) from BarcelonaJS account
+- Tweet meetup event (🚧 create template and link to it) from BarcelonaJS account
```
Comment (dmitry-zaets) [inline guidelines/monthly.md:42]:
```suggestion
- Tweet meetup event (🚧 create template and link to it) from BarcelonaJS account
- Tweet promoting the first speaker (🚧 create template and link to it) from BarcelonaJS account
- Tweet promoting the second speaker (🚧 create template and link to it) from BarcelonaJS account
```

I would suggest doing it in 3 tweets:
 - Meetups announcement
 - Speaker 1 announcement
 - Speaker 2 announcement
This way we would help to promote speakers a bit more

**Comment by onedesert** [comment]
Thanks @dmitry-zaets !

**Review by IagoLast — APPROVED** [review]
LGTM

**Comment by onedesert** [comment]
@IagoLast feel free to add the tweet template as a separate file or at the bottom of this one.

**Pull request merged**